javascript-new/new-objects.js

```diff
@@ -0,0 +1,49 @@
+// ---Object---
+
+function NewObj(value1, value2, value3) {
+  this.key1 = value1;
+  this.key2 = value2;
+  this.key3 = value3;
+}
+
+const anObject = new NewObj(1, 2, 3);
```

Comment:
You should be calling the `Object` constructor rather than creating your own constructor function